### PR TITLE
fix: issue on uncheck category

### PR DESCRIPTION
**Pull request by dikshadhote**
Issue :  Error on uncheck 
#7

### CODE REVIEW

File: src/utils/sort-functions.js [hunk 1]
```diff
@@ -47,9 +47,9 @@ const sortByCategory = (data, stateProduct) => {
   //  sortCategory.includes(item.categoryName): if the selected categories match in data then return those
   const { sortCategory } = stateProduct;
   const sortArray = [...data];
-  if (sortCategory.length != 0) {
-    return sortArray.filter((item) => sortCategory.includes(item.categoryName));
-  }
+  // if (sortCategory.length != 0) {
+  //   return sortArray.filter((item) => sortCategory.includes(item.categoryName));
+  // }
```
Comment (harshitadaswani) [inline src/utils/sort-functions.js:52]: can remove the comments

**Review by harshitadaswani — APPROVED** [review]
- the issue is very well addressed
- can remove the unnecessary comments

**Review by psychoactiv — APPROVED** [review]
refactor looks good can comnit

**Review by ultronjs — APPROVED** [review]
sortReducer is clean and pure.
Logic is working fine now. Good to merge.

**Pull request merged**